default/ship_parts.txt

```diff
@@ -984,6 +984,228 @@ Part
     icon = "icons/ship_parts/bioterm.png"
 
 Part
+    name = "SP_EMP"
+    description = "SP_EMP_DESC"
+    class = Bombard
+    mountableSlotTypes = External
+    buildcost = 5 * [[FLEET_UPKEEP_MULTIPLICATOR]]
+    buildtime = 3
+    location = All
+    effectsgroups = [
+        EffectsGroup    // players can order EMP used on enemies
+            scope = And [
+                Planet
+                Species
+                InSystem id = Source.SystemID
+                OrderedBombardedBy condition = Source
+                Or [
+                    OwnedBy affiliation = EnemyOf empire = Source.Owner
+                    Unowned
+                ]
+                VisibleToEmpire empire = Source.Owner
+                HasTag name = "ROBOTIC"
+            ]
+            effects = [
+                SetPopulation value = Value - 0.5
+                GenerateSitRepMessage
+                    message = "EFFECT_BIOWEAPON"
+                    parameters = [
+                        tag = "system" data = Source.System.ID
+                    ]
+                    empire = Target.Owner
+            ]
+
+        EffectsGroup    // monsters use death spores on player planets
```

Comment:
Then intent was that these could be used by monsters in future.
Comment fixed.  Good catch.